action/action_deserializer.go

```diff
@@ -6,31 +6,15 @@
 
 package action
 
-import (
-	"github.com/iotexproject/iotex-proto/golang/iotextypes"
-)
+import "github.com/iotexproject/iotex-proto/golang/iotextypes"
 
 // Deserializer de-serializes an action
 type Deserializer struct {
```

Comment:
Can this struct be removed?

Reply:
this struct will be used in later PR to de-couple the global `evmNetworkID`, so we keep it